### PR TITLE
Bug 1131678 - [Settings] Update Carets

### PR DESCRIPTION
**Pull request by pivanov**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[pivanov Pavel Ivanov (pivanov)](https://github.com/pivanov) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=02fc957287d3)

**Comment by try-server-hook** [comment]
[pivanov Pavel Ivanov (pivanov)](https://github.com/pivanov) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b9fd704cfeec)

**Comment by try-server-hook** [comment]
[pivanov Pavel Ivanov (pivanov)](https://github.com/pivanov) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=338a39fae6ba)

File: apps/settings/style/icons.css [hunk 1]
```diff
@@ -19,6 +19,7 @@ a.menu-item:after {
   right: 0.3rem;
   font-size: 25px;
   line-height: 6rem;
+  opacity: 0.7;
```
Comment (wilsonpage) [inline apps/settings/style/icons.css]: If you can do this with `color` instead it'll be more performant. Opacity is expensive to paint.

Comment (pivanov) [inline apps/settings/style/icons.css]: I agree but we have :active state too that's why I'm not sure color or opacity ...

Comment (wilsonpage) [inline apps/settings/style/icons.css]: What does the `:active` state look like?

**Comment by try-server-hook** [comment]
[pivanov Pavel Ivanov (pivanov)](https://github.com/pivanov) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=44ace569325f)

File: apps/settings/style/icons.css (hunk 1 above)
Comment (pivanov) [inline apps/settings/style/icons.css]: the light blue list item I will change the opacity with color

**Comment by try-server-hook** [comment]
[pivanov Pavel Ivanov (pivanov)](https://github.com/pivanov) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=642ad9a9ba1c)

**Comment by try-server-hook** [comment]
[pivanov Pavel Ivanov (pivanov)](https://github.com/pivanov) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fec493ee3dc5)

**Pull request merged**